### DISCUSSION TITLE
Macbook notch blues

### DISCUSSION THREAD

**Discussion by bode-p** [jordanbaird/Ice#111] [General]
Is my MacBook's notch obscuring the 'Always-Hidden' section? I don't have this problem on an external display.

**Comment by ChristopherTX15** [comment]
Bartender has the ability to show the hidden items beneath the menu bar so it could extend beyond the notch. It would be great if Ice could implement a similar solution to display those.

**Reply by sandrop** [reply]
100% this. I came here to add this as a feature request

**Reply by joaofrgomes** [reply]
Yep, seconded. The only reason why I figured I could use Ice was the fact that I was using Bartender on my larger devices, the Mac Studio with the dual-monitor setup and the 15'' MBA, mostly to keep the menubar a bit more tidy, but then I realised that if I open a few more apps that spawn their own menu extras, I can't see them, ever (or not without quitting quite a few apps until the former are visible).

I actually own a perpetual Bartender 5 license, so I just downgraded to v. 5.0.48 and will use it until it breaks (or maybe update it and immediately sandbox it with LuLu, who knows, but maybe the next version of macOS will require Bartender 6 and Applause will inevitably turn it into a subscription and leave us in the dust, duh) and wait for Ice to mature and periodically check how development is going.

**Comment by jrdowson** [comment]
On a 16" M3 MacBook Pro (Sonoma 14.5), Ice only displays menubar items that fit to the right of the notch. I'm having a heck of a time getting items to display. I'm a Bartender refugee, and I had hoped Ice might work, but I may have to revert to Bartender 5.0.48. The other alternatives don't appear to be in active development.

**Comment by lume96** [comment]
Hi,
just came here to second this also from my side. I think, like many others, I'm a Bartender refugee and was hoping that ICE would be the go to solution.
I do see that it is on the To Do list "**Display hidden items in a separate bar (e.g. for MacBooks with the notch)**", just wondering if the dev could give some indication regarding the priority of implementation of this?

**Reply by jordanbaird** [reply]
The initial implementation is done. There are a decent number of bugs that need fixing, but I have a beta build that you can download from my latest comment in #1

**Reply by lume96** [reply]
Just found it through Reddit in //.
Thanks sooo much.
